src/services/help-modal-service/help-texts/external-task-token-usage.ts

```diff
@@ -0,0 +1,32 @@
+import {HelpText} from '../../../contracts/index';
+
+export const ExternalTaskTokenUsage: HelpText = {
+  title: 'How to use the token result from previous tasks',
```

Comment:
Dass das über den Token geht, ist vllt. etwas viel Info für den Header.
In erster Linie ist die hier beantwortete Frage ja "Wie kann ich die Ergebnisse von vorherigen FlowNodes verwenden?"

```suggestion
  title: 'How to use the results from previous tasks',
```